src/utilities/theme/role-variants.ts

```diff
@@ -4,13 +4,13 @@ import {RoleVariants} from './types';
 export enum UNSTABLE_Color {
   Surface = '#FAFAFA',
   DarkSurface = '#111213',
-  OnSurface = '#1F2225',
-  Interactive = '#0870D9',
-  Neutral = '#EAEAEB',
+  OnSurface = '#111213',
+  Interactive = '#2E72D2',
+  Neutral = '#111213',
```

Comment:
Why such a dark value for neutral?

Reply:
I'm assuming that these source colours are essentially setting the Hue and that the lightness and saturation are taken care of in the adjustments. Is this incorrect?

Reply:
You're definitely not wrong there. It just felt a bit weird to start with a darker lightness value when all the other inputs essentially start with the light mode value, not the dark mode value.

Reply:
That's fair. The main thing I was trying to do was just make sure all the "greys" in the palette came from the same hue/saturation starting point. We can adjust those down to be a lighter starting point if that keeps things more straightforward.